tests/system_tests_edge_router.py

```diff
@@ -1640,6 +1640,51 @@ def _test_traffic(self, sender, receiver, address, count=5):
             tr.queue.get(timeout=TIMEOUT)
         tr.stop()
 
+    def test_000_check_peer_version_info(self):
+        """
+        Not a link proxy test - ensure router correctly parses peer version
+        numbers advertised in the incoming @open frame
+        """
+        lines = None
+        with open("../setUpClass/INT.A.log") as inta_log:
```

Comment:
Can we use with open(self.router.outdir + '/INT.A.log, 'r') as inta_log:

Reply:
Ah - good catch, thanks!  (fixed)